trillian/docs/ManualDeployment.md

```diff
@@ -329,8 +354,7 @@ script should (mostly) make sense.
 browser should show JSON that indicates an empty tree.
 Alternatively, the `ctclient` command-line tool shows the same information:
 ```bash
-% go install github.com/google/certificate-transparency-go/client/ctclient
-% ctclient --log_uri http://localhost:6966/aramis sth
+% go run github.com/google/certificate-transparency-go/client/ctclient --log_uri http://localhost:6966/aramis sth
```

Comment:
```suggestion
go run github.com/google/certificate-transparency-go/client/ctclient@master get-sth --log_uri http://localhost:8080/testlog
```

You need to specify the version to run, the command is get-sth, and the flag needs to go after the command.